charts/graphiti-mcp/templates/configmap.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "graphiti.fullname" . }}-vars
+  labels:
+    {{- include "graphiti.labels" . | nindent 4 }}
+data:
+  {{- range $key, $value := .Values.vars }}
+  {{ $key }}: {{ $value | quote }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)